packages/adapter-vercel/index.js

```diff
@@ -208,7 +208,7 @@ const plugin = function (defaults = {}) {
 			/** @type {Map<string, string>} */
 			const functions = new Map();
 
-			/** @type {Map<import('@sveltejs/kit').RouteDefinition<import('./index.js').Config>, { expiration: number | false, bypassToken: string | undefined, allowQuery: string[], group: number, passQuery: true }>} */
+			/** @type {Map<import('@sveltejs/kit').RouteDefinition<import('./index.js').Config>, { expiration: number | false, bypassToken: string | undefined, allowQuery: string[] | undefined, group: number, passQuery: true }>} */
```

Comment:
did you intend to change `adapter-vercel` in this PR or is this an accidental commit? if intentional, it probably needs a changeset

Reply:
Oh Shit yeah that was accidental